.buildkite/scripts/make_image.sh

```diff
@@ -53,6 +54,12 @@ case "$BUILD_TYPE" in
         BASE_DOCKER_NAME=$(basename $(pwd))
         DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_${VERSION}"
         ;;
+    "miner-testnet")
+        echo "Doing a testnet miner image build for ${IMAGE_ARCH}"
+        DOCKER_BUILD_ARGS="--build-arg EXTRA_BUILD_APK_PACKAGES=apk-tools --build-arg EXTRA_RUNNER_APK_PACKAGES=apk-tools --build-arg BUILDER_IMAGE=${BUILD_IMAGE} --build-arg RUNNER_IMAGE=${RUN_IMAGE} --build-arg REBAR_BUILD_TARGET=docker_testminer ${DOCKER_BUILD_ARGS}"
+        BASE_DOCKER_NAME=$(basename $(pwd))
+        DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_${VERSION}"
```

Comment:
```suggestion
        DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
```
see the difference between the mainnet and testnet validator builds above between lines 42 and 49; this line is currently identical to the mainnet miner image `DOCKER_NAME` on line 55 so these would likely overwrite each other on quay when they get pushed at the bottom of the file.